migrations/2021-12-14-230218_create_blacklist_community_table/up.sql

```diff
@@ -0,0 +1,18 @@
+-- Your SQL goes here
+
+CREATE TABLE blacklist_community (
+	id serial PRIMARY KEY,
+	reason text,
+	published timestamp not null default now(),
+	creator_id int,
+	community_id int UNIQUE,
+    CONSTRAINT fk_creator
+      FOREIGN KEY(creator_id)
+	  REFERENCES person(id)
+	  ON DELETE SET NULL,
+
+	CONSTRAINT fk_community
+      FOREIGN KEY(community_id)
+	  REFERENCES community(id)
+	  ON DELETE SET NULL
+);
```

Comment:
- It isn't necessary to add a whole new table for something that applies to all communities, then pointlessly join to it. Just add a `hidden` boolean column default false to the `community` table.
- This isn't the syntax we use for table creates. Read over the most recent `up.sql`s in the migration folder for the correct way to do new column or table creates.
- Check out the `mod_ban` table, as you will need a modlog entry for hiding / showing a community. That's where you'll include the reason. Title it `mod_hide_community`, not blacklist.